docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,8 +57,7 @@ If this PR involves code changes, I have:
 - [ ] Tests
   - [ ] Tested the particular change
   - [ ] Figured if the change might have some side effects and tested those as well.
-  - [ ] Smoke tested the extension as such.
-  - [ ] Tested the VSIX built from the PR (so, after you've submitted the PR). You'll find the artifacts by clicking _Show all checks_ in the CI section of the PR page, and then _Details_ on the `ci/circleci: build` test.
+  - [ ] Sanity tested the extension with my changes. 
```

Comment:
Can we add a bit more clarity/guidance here? I can imagine people wondering what exactly this means.

If sanity testing is different depending on the changes made, then we can state that and/or state an example. Like "For example, if the changes are related to jack-in or could affect it, test that jack-in works."

If it's more about testing a set of basic features, maybe we can list them out.

Reply:
We could. I think it's OK if people interpret it differently, though. (But, I don't want them to get confused.)

Reply:
I decided to remove it. It is sort of covered by the remaining items, and CI makes some smoke testing anyway. It's better we ask for specific tests if we start to worry about something in particular in a PR.